slack-clone-compose-sample/src/main/kotlin/io/getstream/slack/compose/ui/util/ChannelUtils.kt

```diff
@@ -0,0 +1,38 @@
+package io.getstream.slack.compose.ui.util
+
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.User
+
```

Comment:
We should review utility methods we provide with our SDK, explain the concepts described below in our docs and give references to them.